SPECS/k3s/k3s-1.25.5.spec

```diff
@@ -82,8 +82,9 @@ exit 0
 %{install_sh}
 
 %changelog
-* Wed Apr 05 2023 Anuj Garg <anujgarg@microsoft.com> - 1.25.5-1
+* Wed Apr 05 2023 Anuj Garg <anujgarg@microsoft.com> - 1.25.5-2
```

Comment:
Please add a new entry.
We do not modify the old entries.

Reply:
Done